<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridSetupWidget</class>
 <widget class="QWidget" name="GridSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="6">
    <widget class="QTableView" name="cellGrid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="chooseTypeLabel">
     <property name="text">
      <string>Cell Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="toggleMeasure">
     <property name="text">
      <string>Toggle Measure Selected</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="addColumnButton">
     <property name="text">
      <string>+ Column</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="chooseType"/>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="columnInt">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSpinBox" name="rowInt">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="toggleStim">
     <property name="text">
      <string>Toggle Stimulate Selected</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="removeColumnButton">
     <property name="text">
      <string>- Column</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="addRowButton">
     <property name="text">
      <string>+ Row</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="removeRowButton">
     <property name="text">
      <string>- Row</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
